test/helpers/cleanup.js

```diff
@@ -1,13 +1,35 @@
 import fs from 'fs'
+import path from 'path'
 
 export function cleanup(path) {
   try {
     const stat = fs.statSync(path)
-    if(stat.isDirectory()) fs.rmdirSync(path)
+    if(stat.isDirectory()) {
+      rimraf(path)
+    }
     else fs.unlinkSync(path)
   } catch(e) { /* swallow exception */ }
 }
 
+/**
+ * Remove directory recursively
+ * @param {string} dir_path
+ * @see https://stackoverflow.com/a/42505874/3027390
+ */
+function rimraf(dir_path) {
+  if (fs.existsSync(dir_path)) {
+    fs.readdirSync(dir_path).forEach(function(entry) {
+      var entry_path = path.join(dir_path, entry);
+      if (fs.lstatSync(entry_path).isDirectory()) {
+        rimraf(entry_path);
+      } else {
+        fs.unlinkSync(entry_path);
+      }
+    });
+    fs.rmdirSync(dir_path);
+  }
+}
```

Comment:
I like this! but we should provide it from `lib`, can you move it there?

Reply:
https://github.com/zeppelinos/zos-lib/pull/148